flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java

```diff
@@ -67,14 +79,14 @@
    * @return an Iceberg catalog loader
    */
   protected CatalogLoader createCatalogLoader(String name, Map<String, String> options) {
-    String catalogType = options.getOrDefault(ICEBERG_CATALOG_TYPE, "hive");
+    String catalogType = options.getOrDefault(ICEBERG_CATALOG_TYPE, ICEBERG_CATALOG_TYPE_HIVE);
     switch (catalogType) {
-      case "hive":
+      case ICEBERG_CATALOG_TYPE_HIVE:
```

Comment:
These changes seem unrelated to loading hive-site.xml. Could you remove them?

Reply:
This is what I think kbendick's suggestion is right. We should extract some constants, but it is unrelated to load hive-site.xml. I will remove them. If necessary ,we can open another PR.